cli_client/python/timesketch_cli_client/commands/sketch.py

```diff
@@ -13,34 +13,137 @@
 # limitations under the License.
 """Commands for sketches."""
 
+import json
 import click
+import pandas as pd
 
 
 @click.group("sketch")
 def sketch_group():
     """Manage sketch."""
 
 
-@sketch_group.command("list")
+@sketch_group.command("list", help="List all sketches. [text,json]")
 @click.pass_context
 def list_sketches(ctx):
     """List all sketches."""
     api_client = ctx.obj.api
+    output = ctx.obj.output_format
+    d = []
```

Comment:
Please rename to something more descriptive

Reply:
done